### ISSUE TITLE
[Feature] [Breakdown] Make highlights smarter

### ISSUE THREAD

**Issue by hyuri** [story-apps/starc#189]
### Issue
Highlights don't seem to store elements added to them.
When we cut and move a piece of highlighted text from one scene to another, the elements associated with that piece of text are not moved/copied with the text.

### Proposal
Make highlights store elements associated with them. And when we cut and paste a piece of highlighted text from one scene to another, the elements should follow them/be moved into the new scene.

**Comment by dimkanovikov** [comment]
Agreed here. Currently, elements added to scenes not to text, that's why them not moved with text itself.
